docs/source/markdown/podman-run.1.md

```diff
@@ -1676,6 +1686,16 @@ $ echo "asdf" | podman run --rm -i --entrypoint /bin/cat someimage
 asdf
 ```
 
+### Setting automatic User Namespace separated containers
```

Comment:
Is this just in run, not create?

Reply:
I think it will work with create as well.  But I wanted to show examples and the userns generated, with the main goal showing they are unigue.

Reply:
Added a create example.